README.md

```diff
@@ -301,6 +301,148 @@ Optional directives for communicating with the internal service. For example `ss
+  haproxy-acme:
+    image: ghcr.io/flobernd/haproxy-acme-tlsalpn01:latest
+    container_name: haproxy-acme-tlsalpn01
+    restart: unless-stopped
+    environment:
+      - "HAPROXY_HTTP_PORT=80"
+      - "HAPROXY_HTTPS_PORT=443"
+      - "HAPROXY_HTTPS_REASSIGN_PORT=8443"
+      - "ACME_MAIL=mail@domain.com"
+      - "ACME_DOMAIN=domain.com"
+      - "ACME_TLSALPN_PORT=10443"
+    volumes:
+      - /docker_data/acme:/var/lib/acme:rw
+    ports:
+      - 80:80
+      - 443:443
+```
+
+### Persistence
+
+It is strongly recommended to specify an external volume for the `/var/lib/acme` directory. Most ACME servers enforce a rate limit for issuing and renewing certificates. If you recreate the container without preserving the internal state of `acme.sh`, a new certificate will also be created each time.
+
+### HAProxy Configuration
+
+The container creates a default configuration file `haproxy.cfg` in the `/usr/local/etc/haproxy` directory.
+
+By mapping the aforementioned path, the primary `haproxy.cfg` can be freely customized. Alternatively, additional configurations can be placed in the `include` directory, which are then loaded after the primary configuration in alphabetical order.
+
+As long as the default config has not been modified or overwritten, the `SERVER_ADDRESS` (*required*), `SERVER_PORT` (default `80`), `HAPROXY_HTTP_PORT` (default `80`), `HAPROXY_HTTPS_PORT` (default `443`), `HAPROXY_HTTPS_REASSIGN_PORT` (default `8443`) environment variables must be set. Otherwise the container will fail to start.
+
+> [!IMPORTANT]
+> When overwriting the default configuration, make sure that the `stats socket` directive is retained. Otherwise, the deployment of certificates will fail.
+
+```
+global
+  # Allow 'acme.sh' to deploy new certificates without reloading
+  stats socket /var/lib/haproxy/admin.sock level admin mode 660
+```
+
+### Environment
+
+#### `ACME_DEBUG`
+
+Set to `1` in order to enable verbose `acme.sh` debug output.
+
+#### `ACME_UPGRADE`
+
+Set to `1` in order to automatically update `acme.sh` to the latest version on container startup. This requires an active internet connection (default: `0`).
+
+#### `ACME_CRON`
+
+Set to `1` in order to enable the certificate renewal cronjob (default: `1`).
+
+#### `ACME_SERVER`
+
+The ACME server to use (default: `letsencrypt`).
+
+Supported values: `letsencrypt`, `letsencrypt_test`, `buypass`, `buypass_test`, `zerossl`, `sslcom`, `google`, `googletest` or an explicit ACME server directory URL like e.g. `https://acme-v02.api.letsencrypt.org/directory`
+
+See also: https://github.com/acmesh-official/acme.sh/wiki/Server
+
+#### `ACME_MAIL`
+
+The mail address for the ACME account registration (*required*).
+
+#### `ACME_DOMAIN`
+
+The domain to issue the certificate for (*required*). To issue a multi-domain certificate (SAN), enter additional domains separated by a space character after the primary domain.
+
+For example: `sub.domain.com` (single), `domain.com *.domain.com` (SAN)
```

Comment:
```suggestion
For example: `sub.domain.com` (single), `domain1.com domain2.com` (SAN)
```

Reply:
I made the same copy/paste mistake for the `http01` image. Going to fix this as well.